src/main/resources/public/src/views/DataRequest.vue

```diff
@@ -118,13 +130,21 @@
             this.$refs.claimerorganization.$data.inputValue = this.organizationSearch
         },
         methods: {
+            handleModelSelected(model) {
+                model.fields.map(field => Vue.util.defineReactive(field, 'requested' , true))
```

Comment:
defineReactive is a function in a tools of vueJS for adding a propety to an vueJS data object